scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -415,7 +425,8 @@ <h5 class="modal-title" id="exampleModalLabel">Edit synopsis</h5>
 
 <script>
 $(function () {
-      function getTerms(query, process) {
+
```

Comment:
When I wrote a gene in Diagnoses->OMIM genes on case page I did not get any suggestions. If I press enter there was the following exception:
```
2020-02-25 14:01:19 MansMacAirDisplay.local flask.app[80431] ERROR Exception on /cust000/643595/diagnose [POST]
Traceback (most recent call last):
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/mansmagnusson/miniconda3/envs/scout36/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/Users/mansmagnusson/Projects/scout/scout/server/blueprints/cases/views.py", line 741, in case_diagnosis
    remove=remove,
  File "/Users/mansmagnusson/Projects/scout/scout/adapter/mongo/case_events.py", line 408, in diagnose
    omim_number = int(omim_id.split(":")[-1])
ValueError: invalid literal for int() with base 10: 'adk'
```

Reply:
That's because the query looks for the diagnoses, not for genes. I'll try to implement the search by genes as well. The app crashing when you type a gene or a non-valid string should be fixed for sure, thanks!